fenix/build.gradle

```diff
@@ -164,7 +166,7 @@ allprojects {
         kotlinOptions.jvmTarget = "17"
         kotlinOptions.allWarningsAsErrors = true
         kotlinOptions.freeCompilerArgs += [
-            "-opt-in=kotlin.RequiresOptIn", "-Xjvm-default=all"
+            "-opt-in=kotlin.RequiresOptIn", "-Xjvm-default=all-compatibility"
```

Comment:
Self-note: this is fine to land because in the follow-up PR, we remove this because we use the new Room version that [has the fixes](https://android-review.googlesource.com/c/platform/frameworks/support/+/2673596).